app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/CreateAccountActivity.java

```diff
@@ -0,0 +1,180 @@
+package com.cmput301f20t21.bookfriends.ui.login;
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+import com.cmput301f20t21.bookfriends.R;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+public class CreateAccountActivity extends AppCompatActivity {
+    private TextInputLayout usernameLayout;
+    private TextInputLayout passwordLayout;
+    private TextInputEditText usernameField;
```

Comment:
These TextInputEditText field can be declared as EditText so you don't have to cast it later.